variables.tf

```diff
@@ -83,4 +83,28 @@ variable "permissions_boundary" {
   description = "ARN of the policy that is used to set the permissions boundary for the role"
   type        = string
   default     = ""
+}
+
+variable "task_cpu" {
+  description = "The ECS Task CPU size"
+  type        = number
+  default     = 256
```

Comment:
Should the default task cpu value be `512` if the default task memory is `1024` per the task memory/cpu combinations here: [aws task definition params](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html)?
The default values for task_cpu and task_memory should be a valid combination.

Reply:
I kept the hardcoded values in the terraform before moving them to variables. Though, I double-checked the values for valid combinations and found the same page. According to the page, a CPU value of 256 can have a memory value of 512MB, 1 GB, or 2GB, as shown in the screenshot below. Is there something there that I am misunderstanding?

![Screen Shot 2022-01-12 at 8 54 47 AM](https://user-images.githubusercontent.com/7129730/149154048-87eb1c7a-f293-4eaa-973c-cbfb306836f7.png)

Reply:
> I kept the hardcoded values in the terraform before moving them to variables. Though, I double-checked the values for valid combinations and found the same page. According to the page, a CPU value of 256 can have a memory value of 512MB, 1 GB, or 2GB, as shown in the screenshot below. Is there something there that I am misunderstanding?
> 
> ![Screen Shot 2022-01-12 at 8 54 47 AM](https://user-images.githubusercontent.com/7129730/149154048-87eb1c7a-f293-4eaa-973c-cbfb306836f7.png)

You are right. Memory size of 1024 is 1 GB, so the hardcoded combination of cpu/memory is correct.